Library/Homebrew/cask/utils.rb

```diff
@@ -104,7 +104,7 @@ def self.method_missing_message(method, token, section = nil)
       message << "during #{section} " if section
       message << "on Cask #{token}."
 
-      opoo "#{message}\n#{error_message_with_suggestions}"
+      ofail "#{message}\n#{error_message_with_suggestions}"
```

Comment:
Makes sense to me but @Homebrew/cask folks may have an opinion about whether we want to keep `opoo`/a warning for users rather than maintainers here?